translation/zh_CN.po

```diff
@@ -0,0 +1,1938 @@
+msgid "Solid infill extruder"
+msgstr "可靠填充挤出头"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1178
+msgid "The extruder to use when printing solid infill."
+msgstr "当打印可靠填充时使用的挤出头。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1184
+msgid "Solid infill every"
+msgstr "每几层可靠填充"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1186
+msgid ""
+"This feature allows to force a solid layer every given number of layers. "
+"Zero to disable. You can set this to any value (for example 9999); Slic3r "
+"will automatically choose the maximum possible number of layers to combine "
+"according to nozzle diameter and layer height."
+msgstr ""
+"该功能使得每指定数目层强制进行一次可靠填充。设为0禁用该功能。可以设为任意值"
+"（如9999）；Slic3r会根据喷嘴直径和层高来自动选择图层的最大可能个数。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1193 xs/src/libslic3r/PrintConfig.cpp:1206
+msgid "↳ solid"
+msgstr "↳ 可靠的"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1197
+msgid ""
+"Set this to a non-zero value to set a manual extrusion width for infill for "
+"solid surfaces. If expressed as percentage (for example 90%) it will be "
+"computed over layer height."
+msgstr ""
+"设为非0值，则表示可靠表面填充的手动挤出宽度。若表示为百分数（如90%），将以层"
+"高为基准计算。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1210
+msgid ""
+"Speed for printing solid regions (top/bottom/internal horizontal shells). "
+"This can be expressed as a percentage (for example: 80%) over the default "
+"infill speed above."
+msgstr ""
+"打印可靠区域的速度（顶部的/底部的/内部水平外壳）。以百分数（如80%）表示，则以"
+"上述的默认填充速度为基准计算。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1221
+msgid "Solid layers"
+msgstr "可靠层数"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1222
+msgid "Number of solid layers to generate on top and bottom surfaces."
+msgstr "在顶部和底部表面生成的可靠层数。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1229
+msgid "Spiral vase"
+msgstr "螺旋式容器"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1231
+msgid ""
+"This feature will raise Z gradually while printing a single-walled object in "
+"order to remove any visible seam. This option requires a single perimeter, "
+"no infill, no top solid layers and no support material. You can still set "
+"any number of bottom solid layers as well as skirt/brim loops. It won't work "
+"when printing more than an object."
+msgstr ""
+"该功能使得在打印单壁物体时会逐渐抬升Z，以便移除可见的缝合点。该选项要求单轮"
+"廓，无填充，无顶部可靠层，无支撑。仍可以设置环边、裙边圈数，和底部可靠层。但"
+"在打印多个物体时不适用。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1236
+msgid "Temperature variation"
+msgstr "温度变化"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1239
+msgid ""
+"Temperature difference to be applied when an extruder is not active.  "
+"Enables a full-height \"sacrificial\" skirt on which the nozzles are "
+"periodically wiped."
+msgstr ""
+"当挤出头处于非活跃状态时用到的温度差。启用一个全高度的环边，纯用于定期给喷嘴"
+"擦拭。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1247 xs/src/libslic3r/PrintConfig.cpp:1256
+msgid "Start G-code"
+msgstr "起始G代码"
+
+# “before extruder has finished heating. If Slic3r detects M104, M109, M140 or M190 in your custom codes, such commands will not be prepended automatically so you're free to customize the order of heating commands and other custom actions. ”
```

Comment:
This is for custom start g-code. If  you do not have M104, M109, M140, or M190 in the box then the text will be inserted after a M190 and M104 that is automatically placed by Slic3r. If the box contains any of those four gcodes the automatic insertion of M190 and M104 will be suppressed and this test will be at the beginning of the gcode.